### ISSUE TITLE
the live2d model is not displayed after the program is started.

### ISSUE THREAD

**Issue by jake123456789ok** [Yang-Xijie/ExARLive2D#22]
The project is deployed normally on ios, but the live2d model is not displayed after the program is started. There is only one button containing xyz settings, and no error is reported during the program running.

Debugging environment:
iphone xr,ios17.3,xcode15.2

**Comment by Yang-Xijie** [comment]
Hi, I just clone the latest codes on my M2 MacBook with Xcode 15.2 and use ⌘R to run this app on my iPad Pro. The model loads successfully.

![IMG_4198](https://github.com/Yang-Xijie/ExARLive2D/assets/57952362/a0625a35-1997-44b7-a6ed-428798cbb649)

You may change the x and y to move the model on different divices (the resolution of an iPhone is different from the iPad). In the app, tap "the info button", open "Settings", change "Zoom" / "X" / "Y".

![IMG_4199](https://github.com/Yang-Xijie/ExARLive2D/assets/57952362/f27112bc-01d1-4aa3-a7b2-7c240aa4958e)

**Comment by Yang-Xijie** [comment]
Before you install the app the first time on your iPhone, you may change the default values in the codes. After you install the app, you can only change these values in the app, because these values are saved locally on your device.

related codes: https://github.com/Yang-Xijie/ExARLive2D/blob/main/ExARLive2D/App/Settings/Settings.swift

**Comment by jake123456789ok** [comment]
Thank you, according to your guidance, the model can be basically displayed after adjusting xyz. The remaining problem is that the model displays anomalies, and it seems that different actions are spliced ​​together. As shown below:
![调试图2](https://github.com/Yang-Xijie/ExARLive2D/assets/48950723/00c63a32-ff4e-4625-b6cf-a0e922175291)

**Comment by Yang-Xijie** [comment]
Yeah... I noticed this problem a long time ago. https://github.com/Yang-Xijie/ExARLive2D/issues/2 However, I don't have sparse time to check and solve it.

**Comment by jake123456789ok** [comment]
> Yeah... I noticed this problem a long time ago. #2 However, I don't have sparse time to check and solve it.

You have done a great job. I will try to re-implement it using Metal and fix this bug in the future. If it succeeds, I will share it as a thank you.
